formats/financial_releases_index/frontend/examples/financial_releases_index.json

```diff
@@ -0,0 +1,35 @@
+{
+  "content_id": "bdd8152b-78b6-479b-a1f8-7bdab36ea68b",
+  "base_path": "/lloyds-share-offer-press-releases",
+  "title": "Financial releases index",
+  "description": "Index page for listing financial press releases",
+  "locale": "en",
+  "public_updated_at": "2015-12-02T17:11:23+01:00",
+  "format": "financial_releases_index",
+  "details": {
+    "image": {
+      "url": "pressrelease.jpg",
+      "alt_text": "lloyds-logo"
+    }
+  },
+  "links": {
+    "press_releases": [
+      {
+        "content_id": "57add221-0376-4f3e-a025-849954df97ac",
+        "title": "Press Release 1",
+        "base_path": "/lloyds-share-offer",
+        "api_url": "",
```

Comment:
The `api_url` is always `https://www.gov.uk/api/content/base_path`. They also always have a description.